### PR TITLE
Update ssh-keygen documentation to rsa 4096

### PR DESCRIPTION
**Pull request by wdpypere**
- update commands
 - update putty screenshots

### CODE REVIEW

File: intro-HPC/ch_account.tex [hunk 1]
```diff
@@ -233,9 +233,9 @@ \subsection{How do SSH keys work?}
 
   You will need to generate a new key pair, when:
   \begin{enumerate}
-    \item  you don't have a key pair yet,
-    \item  you forgot the passphrase protecting your private key,
-    \item  or your private key was compromised.
+    \item  you don't have a key pair yet
```
Comment (itkovian) [inline intro-HPC/ch_account.tex]: Actually, keep the `,` and `.`

File: intro-HPC/ch_account.tex [hunk 2]
```diff
@@ -233,9 +233,9 @@ \subsection{How do SSH keys work?}
 
   You will need to generate a new key pair, when:
   \begin{enumerate}
```
Comment (itkovian) [inline intro-HPC/ch_account.tex]: Should not be an `enumerate`, just an `itemize`.

File: intro-HPC/ch_account.tex (hunk 1 above)
Comment (wdpypere) [reply intro-HPC/ch_account.tex]: I disagree. Either you make a sentence or you make a list. But a list with punctuation is weird, and not proper grammar.

File: intro-HPC/ch_account.tex (hunk 2 above)
Comment (itkovian) [reply intro-HPC/ch_account.tex]: But you can ignore this :)

**Pull request merged**